libfwupdplugin/fu-volume.c

```diff
@@ -725,6 +725,10 @@ fu_volume_new_esp_for_path(const gchar *esp_path, GError **error)
 		if (g_strcmp0(basename, vol_basename) == 0)
 			return g_object_ref(vol);
 	}
+	if (g_file_test(esp_path, G_FILE_TEST_IS_DIR)) {
+		g_info("Using user requested path %s for ESP", esp_path);
```

Comment:
is the info message required? It seems like if the user explicitly set it we should just use it.

Reply:
OK will adjust to debug level.